assets/src/styles/components/VarPicker.scss

```diff
@@ -0,0 +1,119 @@
+.var-picker {
+}
+
+.var-picker * {
+  font-family: Roboto, sans-serif !important;
+  color: black !important;
+  line-height: initial !important;
+}
+
+.var-picker > ul {
+  padding: 0;
+  max-height: 80vh;
+  width: 300px;
+  overflow-y: scroll;
+  resize: 'horizontal';
+  margin-bottom: 0;
+
+  &::-webkit-scrollbar {
+    width: 9px;
+  }
+
+  &::-webkit-scrollbar-track {
+    -webkit-border-radius: 5px;
+    border-radius: 5px;
+    background: rgba(140, 140, 140, 0.2);
+  }
+
+  &::-webkit-scrollbar-thumb {
+    -webkit-border-radius: 5px;
+    border-radius: 5px;
+    background: rgba(140, 140, 140, 0.4);
+  }
+
+  &::-webkit-scrollbar-thumb:hover {
+    background: rgba(140, 140, 140, 0.5);
+  }
+
+  &::-webkit-scrollbar-thumb:window-inactive {
+    background: rgba(140, 140, 140, 0.05);
+  }
+}
+
+.theme-editor-highlight {
+  border: 2px solid yellow !important;
+  box-sizing: border-box !important;
+}
+
+.var-picker .var-group {
+  //max-height: 400px;
+  background: white !important;
```

Comment:
We can remove the commented code, if not needed anymore.

Reply:
> On the general architecture, why is the theme editor on this repo and not on master theme?

Not really a deliberate choice. I think when I started it (over half a year ago) it seemed doing it in master theme would involve a lot of extra set up that was already in place in the blocks plugin. But indeed it makes more sense in master-theme.

It would also make sense as a standalone lib/plugin, since it's not really coupled to WordPress :thinking:

I'll check what moving it in master-theme would involve.